app/views/staff/shared/xml/_budget.xml.haml

```diff
@@ -0,0 +1,5 @@
+%budget{"type" => budget.budget_type, "status" => budget.status}
+  %period-start{"iso-date" => budget.period_start_date.strftime("%Y-%m-%d")}/
```

Comment:
Is it worth declaring the date format in the locale file so we can use it as a formatter, we could then do something like:

`l(budget.period_start_date, format: :iati_date)`

and manage the date format in a single place?

Reply:
I guess we could use a presenter as a solution as well? There is another attribute where a presenter might help, I've not used them before so it may not be a good fit, I'd been keen to learn.

Reply:
I've added the localisation as I'm not sure how to get it working via a presenter, I might revisit later!